apps/backend/src/modules/kobo/tests/kobo.controller.spec.ts

```diff
@@ -0,0 +1,54 @@
+import { AuthService } from '@auth/auth.service';
+import { UserFactory } from '@modules/user/user.factory';
+import { INestApplication } from '@nestjs/common';
+import { Test, TestingModule } from '@nestjs/testing';
+import { AppModule } from '@root/app.module';
+import { DROUGHT, FLOOD, INCIDENT } from '@wfp-dmp/interfaces';
+import request from 'supertest';
+
+import { KoboService } from '../kobo.service';
+
+describe('KoboController', () => {
+  let app: INestApplication;
+  let authService: AuthService;
+  let userFactory: UserFactory;
+  let koboService: KoboService;
+
+  beforeEach(async () => {
+    const testingModule: TestingModule = await Test.createTestingModule({
+      imports: [AppModule],
+    }).compile();
+    userFactory = new UserFactory();
+    authService = testingModule.get(AuthService);
+    koboService = testingModule.get(KoboService);
+    app = testingModule.createNestApplication();
+    await app.init();
+  });
+
+  afterAll(async () => {
+    await app.close();
+  });
+
+  describe('GET - /kobo/n', () => {
+    it('should return 200 with the kobo response', async () => {
+      const koboResult = 'testKoboAnswer';
```

Comment:
better to add some empty 'it' statements now to have a better understanding of what to fill in when the typing is complete

Reply:
will do it in the next ticket